.github/actions/integration-test/action.yml

```diff
@@ -22,7 +22,7 @@ runs:
         export TEST_RUNNER_SERVER_SPECIFICATION_FILE=${{ inputs.test-server }}
         resultsDir=/tmp/results$RANDOM
         echo "Running test suite ${{ inputs.test }} on ${{ inputs.test-server }}"
-        ../gradlew :integration:runTest --args="${{ inputs.test }} $resultsDir" --scan
+        ../gradlew :integration:runTest --args="${{ inputs.test }} $resultsDir" --scan --refresh-dependencies
```

Comment:
I don't think we want to do this; it removes the value of having the dependencies locked in the first place.

Reply:
This is for my testing only. The `--refresh-dependencies` flag will be removed when finalized.

Reply:
This still needs to be removed